src/app/app.component.ts

```diff
@@ -1,10 +1,8 @@
-import { Component, ViewEncapsulation } from '@angular/core';
+import { Component } from '@angular/core';
 
 @Component({
   selector: 'rio-app',
   templateUrl: './app.component.html',
-  // Global styles imported in the app component.
   styleUrls: ['../styles/index.css'],
-  encapsulation: ViewEncapsulation.None,
```

Comment:
Can you show me how to use webpack to load global css into the `<head>` tag? Maybe I can do that in this PR :)